db/schema.rb

```diff
@@ -0,0 +1,86 @@
+# This file is auto-generated from the current state of the database. Instead
+# of editing this file, please use the migrations feature of Active Record to
+# incrementally modify your database, and then regenerate this schema definition.
+#
+# Note that this schema.rb definition is the authoritative source for your
+# database schema. If you need to create the application database on another
+# system, you should be using db:schema:load, not running all the migrations
+# from scratch. The latter is a flawed and unsustainable approach (the more migrations
+# you'll amass, the slower it'll run and the greater likelihood for issues).
+#
+# It's strongly recommended that you check this file into your version control system.
+
+ActiveRecord::Schema.define(version: 20170304172631) do
+
+  create_table "compositions", force: :cascade do |t|
+    t.string   "name"
+    t.text     "notes"
+    t.integer  "map_id",     null: false
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["map_id"], name: "index_compositions_on_map_id"
+  end
```

Comment:
Ah so I don't think we want player\_id here, but we definitely want a user\_id! I forgot about that, haha. That way we can let authenticated users edit their compositions later.

I think I'll leave user\_id nullable for now since there has been talk of letting you save compositions even if you're not logged in and can't edit them later -- see https://trello.com/c/Hh2bx1ll/26-how-can-users-share-the-website-page-app-with-other-people.